### DISCUSSION TITLE
Chat Platform

### DISCUSSION THREAD

**Discussion by egekorkan** [w3c-cg/wot-cg#29] [Polls]
Dear WoT Community Group,

We have been talking for quite a long time about a chat platform to use for casual chats about WoT and also for questions from the community. We have presented some of the prior analyses in the last internal meeting and proposed to adopt [Zulip](https://zulip.com/). We have opened an instance as well at https://wot.zulipchat.com ([invite link](https://wot.zulipchat.com/join/spbvykzvptavkvnzwy2gwayh/)). However, we want to hear your voice as well. All of us are already using one chat platform or the other and it would be good to know if there is a strong tendency towards one, in which case we can simply adopt that.

You can vote or comment below, or join one of the office hour sessions. The reasons for choosing Zulip can be found [here](https://docs.google.com/presentation/d/1RnpC2sG2r37CBkdHPqcJQ_UcEsznGdYxPFAoWv-1RC0/edit?usp=sharing ) but it is mainly driven by the fact that we can have channels, an already good free tier with possibly getting a sponsored standard tier and the open-source nature of the platform.

In the meantime, you can join our Zulip instance and have a look around. We will keep this poll open but decide on the resolution in two-three weeks. An email to point here will be also sent.

From Ege and Cristiano

**Comment by benfrancis** [comment]
I already use IRC, Matrix, Discord and Slack so I would prefer one of those over Zulip.

**Comment by thjaeckle** [comment]
As Gitter.IM will be completely [native Matrix starting next week](https://blog.gitter.im/2023/01/16/gitter-is-going-fully-native-matrix-in-feb-2023/), while still having an easy accessible web interface via https://gitter.im with its benefits (searchable via Google, login via GitHub and other providers, etc.) - I would go with Gitter ;)

**Comment by egekorkan** [comment]
So we think that the vote is concluded and discord is the most preferred option (with a slight margin). We will set up a server and communicate the details at [the next internal meeting on the march 6th](https://www.w3.org/events/meetings/0b0a4962-9dba-4213-903a-582509710914/20230306T170000).

**Comment by egekorkan** [comment]
So our server is available at https://discord.gg/e244zBxSB4 . Feel free to join and suggest new settings/features at the discord-suggestions channel
